samples/FunctionApp/FunctionApp.csproj

```diff
@@ -13,7 +13,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.0.1-preview5" />
```

Comment:
can you share more information here? Don't we expect this app to behave like any function? Are we switching to a project reference?

Reply:
Whoops! This was supposed to not be staged for commit. Undoing.